packages/cli/package.json

```diff
@@ -13,19 +13,32 @@
   "bin": {
     "hops": "commands/hops.js"
   },
-  "main": "index.js",
+  "main": "core.js",
+  "browser": "runtime.js",
+  "server": "runtime.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/xing/hops.git"
   },
   "dependencies": {
+    "@untool/express": "^1.0.0-rc.11",
+    "@untool/webpack": "^1.0.0-rc.11",
     "@untool/yargs": "^1.0.0-rc.11",
-    "enhanced-resolve": "^4.1.0",
-    "find-up": "^3.0.0",
-    "pretty-ms": "^3.2.0",
-    "resolve-cwd": "^2.0.0",
-    "tar": "^4.4.6",
-    "validate-npm-package-name": "^3.0.0",
-    "yargs": "^12.0.1"
+    "hops-express": "11.0.0-rc.49",
+    "hops-mixin": "11.0.0-rc.49",
+    "hops-react": "11.0.0-rc.49",
+    "pretty-ms": "^3.2.0"
+  },
+  "peerDependencies": {
+    "react": "^16.3.0",
```

Comment:
do we want to align the versions with the ones from the `devDependencies`?

Reply:
I've created a separate PR to fix out-of-alignment versions: https://github.com/xing/hops/pull/728